.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -43,22 +43,7 @@ body:
         | This proposal will allow end users to accomplish X | Should |
         | This proposal will allow developers to accomplish Y | Could |
         | This proposal will allow end users to accomplish Z | Won't |
-  - type: input
-    id: files-version
-    attributes:
-      label: Files Version
-      description: Which version of Files are you using?
-      placeholder: v1.5
-    validations:
-      required: true
-  - type: input
-    attributes:
-      label: Windows Version
-      description: Which version of Windows are you using? Press <kbd>Win</kbd> + <kbd>Pause/Break</kbd> and copy.
-      placeholder: e.g. Windows 11 21H2 22000.194
-    validations:
-      required: true
   - type: textarea
     attributes:
-      label: Additional comment
-      description: Add any other comments or screenshots about the feature request here. What device type/UI layout and app version?
+      label: Additional Information
+      description: Anymore info, comments or screenshots about the feature request?
```

Comment:
Not sure if `Anymore` is good here?

Reply:
What about just "Any"?

Reply:
tbh, it'd just leave `Add any other comments or screenshots about the feature request here.` without the other part

Reply:
`Any other comments or screenshots about the feature request?` feels more natural to me, like it's a real person speaking.

Reply:
> `Any other comments or screenshots about the feature request?` feels more natural to me, like it's a real person speaking.

Yeah, but it's kind of informal and *asks the user if they want to add more info* rather than telling them to do so.

Reply:
Well, you are asking them. Not everyone has extra info to add, so this is signifying how you should only add some if you have extra info.

Reply:
@yaichenbaum what's your word on this one?

Reply:
```
label: Comments
description: Additional information, comments or screenshots about the feature request.
```

Reply:
Sounds good to me